### ISSUE TITLE
Mistake commited to stable branch

### ISSUE THREAD

**Issue by amg-web** [instructure/canvas-lms#1835]
Mistake commited to stable branch

https://github.com/instructure/canvas-lms/commit/6684592adfba117b1584067180f7a4db07efe7ce

npx does not have key '--yes' and does NOT require confirmation
check screenshot
![root@elearning_ ~ 25 03 2021 19_11_09](https://user-images.githubusercontent.com/22151881/112514399-f1bcdb80-8d9d-11eb-861e-0d5c00c0d096.png)

**Comment by amireh** [comment]
Starting from NPM 7 it does: 

> To prevent security and user-experience problems from mistyping package names, npx prompts before installing anything. Suppress this prompt with the -y or --yes option.

See https://docs.npmjs.com/cli/v7/commands/npx#compatibility-with-older-npx-versions

**Comment by amg-web** [comment]
I just installed on fresh Ubuntu 18.04
npx in use is 6.14.11
In case you want to use npx --yes <command> you should be sure that v.7 is installed.
But at the moment with v.6 command does not do it's job.

**Comment by amireh** [comment]
Consider updating your version of npm through npm itself:

```
npm install -g npm@latest
```

There aren't any plans to support earlier versions of NPM that I'm aware of. Is there a specific reason that prevents you from upgrading?

**Comment by amg-web** [comment]
No problem to update. But production manual does not tell me anything about NPX update.
I even did not know that NPX is installed. I think it's installed by Nodejs ?

**Comment by amireh** [comment]
That's a good point, I will see to updating the documentation. `npm` and `npx` are part of Node.js, yes. Thanks for reporting.

**Comment by amg-web** [comment]
By the way in documentation:
`If any requested packages are not present in the local project dependencies, then they are installed to a folder in the npm cache, which is added to the PATH environment variable in the executed process. A prompt is printed (which can be suppressed by providing either --yes or --no).`

it means we can see prompt if requested packages are not present in the local project dependencies, is not it?
so if yarn-deduplicate is in project dependency it will not require --yes

**Comment by amireh** [comment]
That is correct. The intent was to not need to install `yarn-deduplicate` as a dependency and only use it on-demand.

**Comment by amg-web** [comment]
anyway I whould remove --yes until npx v7 is installed without extra steps

**Comment by designamx** [comment]
this was an issue for me too, weird thing is that I updated npx to version 10 and still failed, I had to remove the `--yes` option manually from the script
